src/components/action-popover/__snapshots__/action-popover.spec.js.snap

```diff
@@ -0,0 +1,237 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ActionPopover styles renders correctly for the "classic" theme 1`] = `
+.c0 {
+  display: none;
+  margin: 0;
+  padding: 9px 0;
+  box-shadow: 0 5px 5px 0 rgba(0,20,29,0.2),0 10px 10px 0 rgba(0,20,29,0.1);
+  position: absolute;
+  right: 0;
+  background-color: #FFFFFF;
+  z-index: 1;
+  border: 1px solid #C4C6CA;
+  box-shadow: 0 10px 10px 0 rgba(0,20,29,0.1), 0 5px 5px 0 rgba(0,20,29,0.2);
+  padding: 5px 0;
+  right: 10px;
+}
+
+<div
+  className="c0"
+/>
+`;
+
+exports[`ActionPopover styles renders correctly for the "classic" theme 2`] = `
+.c0 {
+  cursor: pointer;
+  box-sizing: border-box;
+  padding: 0 24px;
+  line-height: 40px;
+  white-space: nowrap;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  border: none;
+  width: 100%;
+  color: rgba(0,0,0,0.9);
+  font-size: 14px;
+  font-weight: 700;
+  text-align: left;
+  line-height: 35px;
+  padding: 0 15px 0 10px;
+}
+
+.c0:focus,
+.c0:hover {
+  background-color: #F2F4F5;
+}
+
+.c0:focus {
+  outline: none;
+  box-shadow: inset 0px 0px 0px 2px #FFB500;
+}
+
+.c0:focus {
+  outline: none;
+  box-shadow: inset 0px 0px 0px 2px #255BC7;
+}
+
+<div
+  className="c0"
```

Comment:
This snapshot has the theme object in it so will always need updating if the theme is altered